### DISCUSSION TITLE
Xarray Community Developer position @ Earthmover PBC

### DISCUSSION THREAD

**Discussion by dcherian** [pydata/xarray#9059] [Announcements]
Hello all,

We are looking for a Xarray community developer to boost adoption of Xarray in the biomedical space.

> This role will focus on building connections, both technical and social, between the Xarray project and the biomedical research community. We want you to join the Earthmover team to do a mix of technical engineering work and community stewarding, serving the needs of biomedical researchers in fields like microscopy, genomics, and neurophysiology.

Please see the ad [here](https://jobs.gusto.com/postings/earthmover-xarray-community-developer-498dca94-335e-4d5c-a6c7-83ca19772512) (and please share widely).

Earthmover values a diverse workplace and **strongly** encourages women, people of color, LGBT individuals, people with disabilities, members of ethnic minorities, foreign-born residents, and veterans to apply. We want you.

The expected salary range for this position is $135,000 - $150,000 per year.

Please reach out if you have questions: [deepak@earthmover.io](mailto:deepak@earthmover.io)

Deepak

---

Deepak Cherian
Forward Engineer
Earthmover PBC

**Comment by dcherian** [comment]
cc @ilan-gold  @ivirshup @nvictus @asford @jni @tlambert03 @GenevieveBuckley

**Reply by GenevieveBuckley** [reply]
Looks great, I'd apply to it if it weren't for the US work authorisation. Unfortunately most of my contacts are Australian, but maybe @jsignell, @InessaPawson or @hugobowne have other ideas.

**Reply by GenevieveBuckley** [reply]
And @martindurant has some medical imaging research in his background, maybe he knows a few people with software & biomed experience who could be interested

**Reply by martindurant** [reply]
Thanks for the ping @GenevieveBuckley . My medical contacts were all Canadian and very much more clinical research rather than big data. Also, several years have passed :| .

**Comment by hugobowne** [comment]
i'd happily share a LinkedIn post or tweet if someone has shared on these platforms?

**Reply by jhamman** [reply]
Thanks @hugobowne! Here are two options:

- https://x.com/_jhamman/status/1796676382543757329
- https://www.linkedin.com/posts/activity-7205888424564727808-VhBE?utm_source=share&utm_medium=member_desktop

**Reply by hugobowne** [reply]
done!

https://x.com/hugobowne/status/1800342341003427918

https://www.linkedin.com/posts/hugo-bowne-anderson-045939a5_earthmover-activity-7206108256832819200-XuKw?utm_source=share&utm_medium=member_desktop

**Reply by jhamman** [reply]
Thanks @hugobowne!

**Reply by nvictus** [reply]
Done as well!

**Reply by InessaPawson** [reply]
I've reposted as well. 
P.S. Excited to see that community building work in open source is being funded!
